nginx.conf

```diff
@@ -0,0 +1,54 @@
+# nginx.conf 
+
+user  nginx;
+worker_processes 1;
+
+error_log  /var/log/nginx/error.log debug;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+        '$status $body_bytes_sent "$http_referer" '
+        '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    upstream app {
```

Comment:
Calling this `app` is a bit mysterious... Can we name this `auth` as well, or would that be too many things named auth...?

Reply:
I've changed its name to `github_auth`